automation/run-integration-tests.sh

```diff
@@ -85,13 +85,22 @@ function run_exit {
     remove_container
 }
 
+function open_shell {
+    res=$?
+    [ "$res" -ne 0 ] && echo "*** ERROR: $res"
+    set +o errexit
+    docker_exec 'echo "pytest tests/integration --pdb" >> ~/.bash_history'
+    docker_exec 'cd /workspace/nmstate; exec /bin/bash'
+    remove_container
```

Comment:
It will be safer to leave run_exit (in case it will change in the future).
Or do you see a problem with this?

Reply:
No, this was just because I assumed that the artifacts collection is not needed for manual runs. But it does not hurt to keep it.